metrics/web/lib/analytics/domain/usecases/parameters/user_id_param.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:meta/meta.dart';
+
+/// A class that represents the user identifier parameter.
+@immutable
+class UserIdParam {
```

Comment:
It seems like this class is a duplicate of the `auth/domain/usecases/parameters/user_id_param.dart`. I think we can just move this class to the `common/domain/usecases/parameters` package to use it in `auth` and `analytics` modules.